tracer/src/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Source generators always must target netstandard2.0, so adding this extra target doesn't do anything